### ISSUE TITLE
Support for both fileUri and contentUri

### ISSUE THREAD

**Issue by wodCZ** [argo22com/react-native-tus-client#1]
As of now, we create [TusAndroidUpload](https://github.com/argo22com/react-native-tus-client/blob/master/android/src/main/java/com/vinzscam/rntusclient/RNTusClientModule.java#L160-L162) which only accepts Content URIs.

In the last release of [react-native-image-picker](https://github.com/react-native-image-picker/react-native-image-picker) both File URI and Content URI can be returned, which causes crashes.

A detection of Upload class to be used based on the URI schema could be done:
```java

            Uri uri = Uri.parse(fileUrl);
            Log.d(TAG, "Uri: " + uri);


            if (uri.getScheme().equals(ContentResolver.SCHEME_CONTENT)) {
                Log.d(TAG, "Creating TusAndroidUpload");
                upload = new TusAndroidUpload(uri, reactContext);
            } else {
                Log.d(TAG, "Creating generic TusUpload");
                upload = new TusUpload(new File(fileUrl));
            }

            upload.setMetadata(metadata);

```
but it's currently failing due to (I suppose) permission errors.

Related issue: https://github.com/react-native-image-picker/react-native-image-picker/issues/1656

**Comment by ravirajn22** [comment]
Instead of using 
`new File(String pathname)`
use 
First convert the stringUrl to Uri object and use it
`File(URI uri)`

This should work

**Comment by wodCZ** [comment]
No way 🤦‍♂️ the File constructor didn't want the `android.net.Uri`, so I went with the string uri, but with `URI javaUri = URI.create(fileUrl);` from `java.net.URI` it works as expected.

Thank you @ravirajn22! I owe you :)

**Comment by wodCZ** [comment]
Closed in 4511aadd7def6d87e3b221323acf89feac357492
